src/input.rs

```diff
@@ -153,6 +154,8 @@ impl<'a> Input<'a> {
     /// This function will block until the user submits the input. If the user cancels the input,
     /// an error of type `io::ErrorKind::Interrupted` is returned.
     pub fn run(mut self) -> io::Result<String> {
+        let ctrlc_handle = ctrlc::show_cursor_after_ctrlc(&self.term)?;
```

Comment:
this approach seems to work more or less but having to close the handle is a bit cumbersome though. it seems a bit safer then using unsafe operations to register/unregister. signal_hooks also probably won't work on Windows as per docs.

@jdx any ideas?

Reply:
can you even hide the cursor to begin with on windows?

Reply:
I'll test real quick on my windows laptop

Reply:
Windows support for signals seems hopeless at this point. The `ctrlc` crate goes a long way to make it happen but theres no way to unregister for our use-case. For now i guess we have to accept this as a known issue.